src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/main.py

```diff
@@ -87,24 +95,17 @@ def update_deployment_account_output_parameters(
     deployment_account_parameter_store = ParameterStore(
         deployment_account_region, deployment_account_role
     )
-    parameter_store = ParameterStore(
-        region, deployment_account_role
-    )
+    parameter_store = ParameterStore(region, deployment_account_role)
     outputs = cloudformation.get_stack_regional_outputs()
     kms_and_bucket_dict[region] = {}
-    kms_and_bucket_dict[region]['kms'] = outputs['kms_arn']
-    kms_and_bucket_dict[region]['s3_regional_bucket'] = (
-        outputs['s3_regional_bucket']
-    )
+    kms_and_bucket_dict[region]["kms"] = outputs["kms_arn"]
+    kms_and_bucket_dict[region]["s3_regional_bucket"] = outputs["s3_regional_bucket"]
     for key, value in outputs.items():
         deployment_account_parameter_store.put_parameter(
-            f"cross_region/{key}/{region}",
-            value
-        )
-        parameter_store.put_parameter(
-            f"cross_region/{key}/{region}",
-            value
+            f"cross_region/{key}/{region}", value
         )
+        parameter_store.put_parameter(f"cross_region/{key}/{region}", value)
+        parameter_store.put_parameter(f"/cross_region/{key}/{region}", value)
```

Comment:
Shouldn't we prefix ADF as we do for all of ADFv4 params?

Reply:
yes, you're right. Good catch!

Reply:
This is actually already done by the parameter_store class. (L100)